accounts/keystore/key.go

```diff
@@ -179,6 +179,8 @@ func storeNewKey(ks keyStore, rand io.Reader, auth string) (*Key, accounts.Accou
 		zeroKey(key.PrivateKey)
 		return nil, a, err
 	}
+	fmt.Printf("Your new key is generated. Please backup the key file: %s\n", a.URL)
```

Comment:
This is used from other locations aswell, we shouldn't `fmt.Printf` from within a lower library. If you modify `StoreKey` instead, which is only called from `accountCreate`, then you can pick out the URL from the returned `account`, and either do the printout there, or -- better, return the entire `account` instead of just the address

Reply:
I returned the entire `account` instead of just the address.